CONTRIBUTING.md

```diff
@@ -95,9 +95,10 @@ changed and why. Follow these guidelines when writing one:
 1. The first line should be 50 characters or less and contain a short
    description of the change. All words in the description should be in
    lowercase with the exception of the ones that refer to code (e.g.
-   function/variable names). The description should be prefixed with the name
-   of the changed subsystem and start with a present-tense or imperative verb
-   (e.g. "net: add localAddress and localPort to Socket").
+   function/variable names), proper nouns, and acronyms. The description should
+   be prefixed with the name of the changed subsystem and start with a
+   present-tense or imperative verb (e.g. "net: add localAddress and localPort
```

Comment:
I now favor just "imperative verb" and dropping "present-tense" but if anyone (@gibfahn or otherwise) feels that including `present-tense` is better, that's OK too.